### ISSUE TITLE
GitMate Labelled Closed Issues Tonight

### ISSUE THREAD

**Issue by sils** [TEdit/Terraria-Map-Editor#1044]
Hi everyone :)

TLDR: [GitMate.io](https://gitmate.io) acted on closed issues (and PRs) unneededly tonight, likely on your repo. The issue is fixed now and we're improving QA. More info at https://gitlab.com/gitmate/open-source/gitmate-2/issues/275.

You're getting this email because you have been affected by a bug in GitMate.io this night and you're using the feature that was buggy.

In short: GitMate's stale rules acted on closed issues and PRs. That might have been comments like "this issue will be closed now because it's inactive" or the addition of stale labels on already closed issues.

We are sorry that this happened and fixed the problem as soon as we were aware. We are also introducing new measures to improve our stability with a new QA stage in addition to our existing ones so those kinds of issues don't happen again.

Please let us know if you would like to get a script to revert those actions; it seems though as most communities would rather keep it than sending out additional notifications.

We've summarized the incident at https://gitlab.com/gitmate/open-source/gitmate-2/issues/275 .

Please reach out to us if you have any further questions about this.

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
Founder of http://coala.io/, http://viperdev.io/ and http://gitmate.io/

**Comment by BinaryConstruct** [comment]
Thanks for the update. It shouldn't cause any problems here.
